2-mandatory/1-weather-report.js

```diff
@@ -12,9 +12,29 @@
 */
 
 function getTemperatureReport(cities) {
-    // TODO
+  // TODO
+  return cities.map(city => `The temperature in ${city} is ${temperatureService(city)} degrees`)
+
 }
 
+// console.log(getTemperatureReport(["London", "Paris", "São Paulo"]));
```

Comment:
Bit too much commented out code here but the tests pass and the non commented out code looks good, so good work